config/sync/core.entity_form_display.paragraph.text_media.default.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
     - field.field.paragraph.text_media.field_caption
     - field.field.paragraph.text_media.field_caption_display
     - field.field.paragraph.text_media.field_credit
+    - field.field.paragraph.text_media.field_image
```

Comment:
We've had a call with the DAM people and the DAM has a process (details pending) or creating a thumbnail/placeholder image from the uploaded video that we could use the API to capture and store against the media item.

This method of allowing the user to select a specific image as the placeholder will give them more freedom but be more of admin burden. @endre-soo-TPX do you have any opinions on which way would be best? I'd lean towards just using the approach Luke has already put in place to same time and effort.